code/modules/clothing/under/jobs/engineering.dm

```diff
@@ -1,3 +1,10 @@
+//Fits all uniforms to Vox
+/obj/item/clothing/under/rank/

```

Comment:
So what you're saying is if I defined obj/item/clothing/under/

then did the whole species_fit sprite_sheets business in another file, it'll apply here?